mcp/tools/jenkins.py

```
@@ -504,3 +506,109 @@ async def jenkins_whoami(
+                            "description": desc,
+                            "default": default,
+                            "choices": choices,
+                        })
+        # 2️⃣ If JSON fails or no params, fallback to XML
+        if not params:
+            xml_resp = await client.get(f"{build_job_path(jobFullName)}/config.xml")
+            if xml_resp.is_success:
+                params = _parse_config_xml(xml_resp.text)
+
+        return {"output": json.dumps(params), "error": False}
+    except Exception as e:
```

Ruff (0.14.0)

Do not catch blind exception: `Exception`

(BLE001)